nuxt.schema.ts

```diff
@@ -87,10 +87,17 @@ export default defineNuxtSchema({
           /**
            * Base URL for sitemap. All links in sitemap will be relative to this URL.
            *
-           * @example 'https://exactpro.github.io/docs-toolkit'
+           * @example 'https://exactpro.github.io'
            * @type {?string}
            */
-          baseUrl: undefined
+          baseUrl: undefined,
```

Comment:
`baseUrl` have misleading name. It should be `hostname`. 

You still have option to delete prefix from config and extract it from `baseUrl`. Moreover, I think it is better for user to fill only 1 option instead of 2.

Reply:
tried replacing baseUrl with hostname, but this is not read in sitemap.xml

Reply:
As option is changed in config, you also should change it whenever it is used.

Reply:
Fixed !